chapter_3/embedded_ansible_using_awx.md

```diff
@@ -1,21 +1,68 @@
 ## Embedded Ansible using AWX
 
-- Background
-  - Embedded Ansible in Fine Release
-  - Ansible Tower
-  - Setup Embedded Ansible (http://talk.manageiq.org/t/howto-setup-embedded-ansible/2291/2) (Do not write whole procedure)
-- AWX intro
-- How it is integrated in ManageIQ
-  - 4 Docker containers
-  - Watch for awx logs
-  - Access awx bash and look for errors
-- No problem of Ansibe Tower license
-
 ### Background
 
-In ManageIQ Fine release, it is configured to use [Ansible Tower](https://www.ansible.com/products/tower) for automation. Integrating Ansible Tower in ManageIQ requires a tower license.
+In ManageIQ Fine release, it is configured to use [Ansible Tower](https://www.ansible.com/products/tower) for Embedded Ansible role. Integrating Ansible Tower in ManageIQ requires a tower license.
 
-| Info | Procedure to setup Embedded Ansible in ManageIQ Fine is described in [Setup Embedded Ansible](http://talk.manageiq.org/t/howto-setup-embedded-ansible/2291) |
+| Info | Procedure to setup Embedded Ansible in ManageIQ Fine version is described in [Setup Embedded Ansible](http://talk.manageiq.org/t/howto-setup-embedded-ansible/2291) |
 |------|:------|
 
 In Gaprindashvili release of ManageIQ, Ansible Tower is replaced by AWX for implementation of Embedded Ansible. [AWX](https://github.com/ansible/awx) is open source project around the Ansible Tower codebase. Unlike Ansible Tower, AWX doesn't requires license to be added for enabling Embedded Ansible in ManageIQ.
+
+### AWX inside ManageIQ Appliance
+
+AWX is deployed as group of containers inside ManageIQ appliance. EVM manage awx containers using the `docker-api` gem. When Embedded Ansible role is enabled on an appliance, it pulls latest awx docker images and run the containers.
+
+To enlist docker images, type `docker images` in the terminal of ManageIQ appliance.
+
+```
+[root@localhost ~]# docker images
+REPOSITORY                   TAG                 IMAGE ID            CREATED             SIZE
+docker.io/ansible/awx_task   latest              e45b49c04679        4 days ago          1.017 GB
+docker.io/ansible/awx_web    latest              2340b2955aa4        4 days ago          989.5 MB
+docker.io/memcached          alpine              5c28baaba1e0        13 days ago         7.871 MB
+docker.io/rabbitmq           3                   02c3edbcf3d1        3 weeks ago         126.9 MB
+```
+
+These four docker images are necessary to start awx properly.
+AWX docker containers are linked together. You can view the running containers using `docker ps` command, as follows:
```

Comment:
can you please convert the sentence to Active Voice i.e "You can view....command," to "To view the running containers use `docker ps` command."